backend/core/src/listings/listings.service.ts

```diff
@@ -148,9 +148,9 @@ export class ListingsService {
     const qb = getView(this.listingRepository.createQueryBuilder("listings"), view).getViewQb()
     const result = await qb
       .where("listings.id = :id", { id: listingId })
-      .orderBy({
-        "listingPreferences.ordinal": "ASC",
-      })
+      // .orderBy({
```

Comment:
@seanmalbert Quick q for you here on why this would have been here before?

Reply:
It ordered the preferences, so we could keep it for listingMultiselectQuestions unless we're ensuring the order somewhere else that's more efficient.

Reply:
We do sort elsewhere, and now we'd have to sort across application sections here, so I'll move forward by removing it from here 👍